### ISSUE TITLE
3rd Birthday?

### ISSUE THREAD

**Issue by SvenCakemann** [marco-calautti/Rainbow#26]
I'm seeing that the tool is supposedly compatible with 3rd Birthday DAT, but exporting my game from the UMD I'm not seeing any DAT files. As far as  I knew there was the PKG file that can get extracted with Noesis, but that was kind of it. Are custom textures for characters/UI and such supported or am I just misunderstanding what is being described here?

**Comment by marco-calautti** [comment]
You need first to extract the individual files from the game archive in order to open them with rainbow. I am not sure whether there is a publicly available tool for that.

**Comment by SvenCakemann** [comment]
Thanks for the reply. If you have both 3rd.pkg and 3rd.fsd together in a folder, Noesis can extract the individual files out. (examples: arc/file0016.t3bpack, audio/file0031.sscf, video/file0275.psmf)

Can you give me an example of a file that Rainbow can open? At least from my testing the t3bpack files (models, textures, some sounds together) cannot be opened and there _are_ no "dat" files.

**Comment by marco-calautti** [comment]
Yeah, actually this dat extension is quite generic, and it is the extension another unpacker gives to certain files. I thought this unpacker was publicly available. It seems noesis gives different names.

From what I remember, some of these graphics are themselves further enclosed in some subpackages called packs. Others are already found in the root of the main PKG archive. The latter graphics should be the 323rd, 324th and 325th file in the list (I can't remember whether the numbers start from 0 or 1)

**Comment by SvenCakemann** [comment]
Ah okay. Do you recall which tool it was? If not, if it's out there I can hopefully find it.

**Comment by marco-calautti** [comment]
I don't think it had a specific name. It was made by Vash. Anyway i guess you can still use noesis. It seems it also numbers the files in increasing order. Simply change, e.g., the 324th file extension to dat and open it with Rainbow.
